src/SwedbankPay.Sdk.Infrastructure/Consumers/ConsumersResource.cs

```diff
@@ -64,7 +72,7 @@ public Task<Consumer> InitiateSession(ConsumersRequest consumersRequest)
         /// <returns></returns>
         public Task<BillingDetails> GetBillingDetailsInternalAsync(Uri url)
         {
-            return this.httpClient.GetAsJsonAsync<BillingDetails>(url);
+            return this.HttpClient.GetAsJsonAsync<BillingDetails>(url);
```

Comment:
`this.` should only be used for private fields in the current class. If we're referencing a property, drop it and if we need to explicitly reference the base class to avoid naming collisions, use `base.`.

```suggestion
            return HttpClient.GetAsJsonAsync<BillingDetails>(url);
```